real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -26,6 +26,23 @@
 	// "C:\\Users\\abc\\buggy-log\\VisionSystem.exe";
 
 	public static String LOG_FILE_LOCATION = "logs";
+	public static String LOG_STOP_MESSAGE = "STOP_LOGGING";
+	public static String SW_VERSION_HIGHLEVEL = "1.0.0";
+	
+	
+	//names of the various sensors
+	public static String SENSOR_NAME_BRAKE = "brake";
+	public static String SENSOR_NAME_CAMERA = "camera";
+	public static String SENSOR_NAME_ENCODER = "encoder";
+	public static String SENSOR_NAME_GPS = "gps";
+	public static String SENSOR_NAME_GUI_LOGGING_BUTTON = "logging_button";
+	public static String SENSOR_NAME_IMU = "imu";
+	public static String SENSOR_NAME_STEERING = "steering";
+	
+	//other message types
+	public static String MESSAGE_TYPE_RESET = "reset";
+	public static String MESSAGE_TYPE_STATUS = "status";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1273020439/issues/source?bid=2938665&fileBranchId=2938743#l44)